### PR TITLE
Update pipeline-service prod

### PR DESCRIPTION
**Pull request by enarha**
The change includes the Tekton Results update to enable TLS verification (pipeline-service commit SHA 706387e06fde062c3c8fc23531ea9fe8a62a372f), but does not include the OSP version bump (pipeline-service commit SHA 8eada62e76fb135d2cba5b7929411181c0a0e9e0). The latter is planned for update a week from now.

### CODE REVIEW

File: components/pipeline-service/production/stone-prd-m01/deploy.yaml
Comment (gabemontero) [inline components/pipeline-service/production/stone-prd-m01/deploy.yaml:1471]: yep we are allowing a bump of the api server image while we still pin the watcher image at the level with the mem leak (and the performance cheats which we need to live with until the WG approved rewrite of logging to bypass the GRPC calls occurs).

the watcher image is not changed here

looks good

File: components/pipeline-service/production/stone-prd-m01/deploy.yaml
Comment (gabemontero) [inline components/pipeline-service/production/stone-prd-m01/deploy.yaml:1777]: is this intentional ?? .... I think konflux-ci is the new approved way ... maybe this is getting picked up from the pipeline service repo and should be reverted there ?

File: components/pipeline-service/production/stone-prd-m01/deploy.yaml
Comment (gabemontero) [inline components/pipeline-service/production/stone-prd-m01/deploy.yaml:1886]: yeah pretty sure @amisstea recently enabled this .... something is inadvertently reverting this @enarha

**Review by gabemontero — CHANGES_REQUESTED** [review]
the pinning looks good @enarha but found a couple of anomalies   ... ptal

File: components/pipeline-service/production/stone-prd-m01/deploy.yaml
Comment (enarha) [reply components/pipeline-service/production/stone-prd-m01/deploy.yaml:1777]: Good catch. I compared that with staging and considered it good, but it isn't. The problem is #3848 updated the deploy.yaml files directly, instead of the refs in the pipeline-service repo and now our update is overriding it. I'll create the fix in the service repo, but we can't consume it without the OSP bump (or at least without revering it in both service and infra repos and staging clusters, which I'm not willing to do). So I'll create the change in the service repo and revert those specific changes in the deploy.yaml files. @rcerven FYI.

File: components/pipeline-service/production/stone-prd-m01/deploy.yaml
Comment (enarha) [reply components/pipeline-service/production/stone-prd-m01/deploy.yaml:1886]: Same thing as above, different PR #3818 , changes the deploy.yaml files instead of the pipeline-service repo. I'll revert the changes to the deploy.yaml files and create a PR to update pipeline-service. @Omeramsc FYI.

File: components/pipeline-service/production/stone-prod-p02/deploy.yaml
Comment (enarha) [inline components/pipeline-service/production/stone-prod-p02/deploy.yaml:1347]: It looks like this cluster was not up to date, thus the change.

File: components/pipeline-service/production/stone-prod-p02/deploy.yaml
Comment (enarha) [inline components/pipeline-service/production/stone-prod-p02/deploy.yaml:2200]: Same as above. It is inline with the other clusters though.

File: components/pipeline-service/production/stone-prod-p02/deploy.yaml
Comment (gabemontero) [reply components/pipeline-service/production/stone-prod-p02/deploy.yaml:1347]: yep this is OK and lines up with https://github.com/redhat-appstudio/infra-deployments/blob/main/components/pipeline-service/staging/stone-stg-rh01/deploy.yaml#L1349-L1353

**Review by gabemontero — APPROVED** [review]
/hold
(until you release the hold at the date you specified in your konflux announce email
/lgtm

**Comment by gabemontero** [comment]
I think you kube-linter error is a github.com hiccup
```
Error when running kustomize build for components/pipeline-service/staging/base: Error: accumulating resources: accumulation err='accumulating resources from 'https://github.com/openshift-pipelines/pipeline-service.git/operator/gitops/argocd/pipeline-service?ref=8eada62e76fb135d2cba5b7929411181c0a0e9e0': URL is a git repository': failed to run '/usr/bin/git fetch --depth=1 https://github.com/openshift-pipelines/pipeline-service.git 8eada62e76fb135d2cba5b7929411181c0a0e9e0': fatal: unable to access 'https://github.com/openshift-pipelines/pipeline-service.git/': Failed to connect to github.com port 443 after 21 ms: Connection refused
```

**Comment by enarha** [comment]
> /hold (until you release the hold at the date you specified in your konflux announce email /lgtm

This change does not include the OSP bump, see the commit message. I actually planned to merge it as soon as the CI pass and I get the respective approvals. I see the CI is already green, so unless you have any objections, I'll remove the hold.

**Comment by gabemontero** [comment]
> > /hold (until you release the hold at the date you specified in your konflux announce email /lgtm
> 
> This change does not include the OSP bump, see the commit message. I actually planned to merge it as soon as the CI pass and I get the respective approvals. I see the CI is already green, so unless you have any objections, I'll remove the hold.

sorry totally missed that in the description

/hold cancel

**Pull request merged**
